### ISSUE TITLE
Tiefling variant traits mislabelling spells

### ISSUE THREAD

**Issue by BlackRose-XIII** [Sklore/HL_DD_5e_Colab#973]
Kind of complicated to explain but I'll do my best. A number of the Tiefling variant racial traits, specifically the ones connecting them to different layers of hell (i.e. Mephistopheles, Zariel, Levistus), were copied, and some of the spell-like abilities are being named "Tenser's Floating Disc" instead of the appropriate spell name. Spell description is being displayed correctly.

As a side note, the "Legacy of Avernus" ability for Zariel's Tieflings specifies that "Branding Smite" is cast as a 3rd level spell, but this is missing from the description text. Ditto "Legacy of Cania" with Flame Blade, and possibly other I didn't see.

**Comment by BlackRose-XIII** [comment]
Additionally, variant racial traits which alter the racial stat bonuses are affecting stats correctly but the changes are not noted in the description text

**Comment by BlackRose-XIII** [comment]
Updated the descriptions as necessary, submitted pull request. Once someone fixes the issue with the spell names this can be closed

**Comment by Fenris447** [comment]
It looks like you may have pasted your updates in without removing what they’re replacing. I think you may have actually done the whole file haha.

There’s an option on GitHub for “Files Changed”, which normally will show you the old lines in red and the new lines in green. When I look at your pull request, it shows a giant block of green and no red; IE a bunch being added and nothing removed.

**Comment by BlackRose-XIII** [comment]
Sorry, I thought I was following the directions in the How-to correctly, but maybe I misunderstood something. It told me to:
"Open the file you edited within the Hero Lab Editor in Notepad (personally I prefer Notepadd++) and copy the entire contents of the file.
Paste the contents of the file into the GitHub edit file screen replacing all other content."

I can seem to figure out how to cancel the pull request, and I don't seem to be able to start a new branch, I'm assuming because the pull request is still pending? I'm not really sure of a lot of things but I think I'm stuck now.

**Comment by BlackRose-XIII** [comment]
After a significant amount of head scratching, I think I figured out a better way to do everything. I've submitted a new pull request, please tell me if that's better.

**Comment by Fenris447** [comment]
I’ll take a look when I have a second. Don't worry, it’s all a learning experience. I’ll cancel out anything that needs to be cancelled and tweak anything that needs to be tweaked.

I really do appreciate the effort! It’s been forever since anyone else was able to contribute to the pack, so I’m just excited to have help in any form or amount.

**Comment by Fenris447** [comment]
Now that I’m looking at the file further, it looks like there are more copy and paste issues than we initially realized. I’m not sure any of them actually effect the functionality, but I’m going to dig in when I have the chance. 

Each Legacy of ___ thing seems to be copied from the Legacy of Maladomini. And all the bootstrap conditions on each Legacy trait are testing to see if ra5CLegMal has been replaced. Instead, Legacy of Dis should be testing to see if Legacy of Dis has been replaced, not testing if Legacy of Maladomini has been replaced. It’s an easy enough fix.

The other thing I want to do is go in and add an sNameMod field to any of the spells that are cast at a higher level than normal. So for the Baalzebul tief, I wanna add an sNameMod onto Ray of Sickness that says “2nd-level”.

I can do all that. But if you wanted to give it a shot, let me know.

**Comment by Fenris447** [comment]
So I took what you did and made further adjustments based on what I had found. It looks like someone has copied and pasted a lot of the code, including renaming Floating Disc (which is what it’s called in the pack) to Tenser’s Floating Disc, and that rename carried over to the pasted abilities. Fixed that.

I double checked the book text and tweaked where you had put some things as being upcast where they’re not, and added name modifiers to the spells that *are* being upcast. 

I don’t think I can change your pull request, so I’m going to cancel it and upload the version with both our sets of changes to my branch. Thanks again for the work you did!
